src/assets/styles/base.scss

```diff
@@ -277,6 +277,10 @@ p {
   background-color: $gray1 !important;
 }
 
+.dk-gray-background {
+  background-color: $gray3 !important;
+}
```

Comment:
I put this here because it seems like the kind of general thing we'd want to reuse.